PWGHF/TableProducer/candidateCreator2Prong.cxx

```diff
@@ -653,7 +653,15 @@ struct HfCandidateCreator2ProngExpressions {
   Produces<aod::HfCand2ProngMcRec> rowMcMatchRec;
   Produces<aod::HfCand2ProngMcGen> rowMcMatchGen;
 
+  // Configuration
+  o2::framework::Configurable<bool> rejectBackground{"rejectBackground", false, "Reject particles from background events"};
+  o2::framework::Configurable<int> centralityEstimator{"centralityEstimator", 0, "Centrality estimator: 0 - FT0C, 1 - FT0M"};
```

Comment:
This should be a template argument and you should have a process function for each estimator. To do this, you can move what is in the `processMc` function in a dedicated template function, to be called inside the different process functions